src/app/entry/delete/dialog/delete-entry-dialog.component.html

```diff
@@ -0,0 +1,22 @@
+<h1 mat-dialog-title>Are you sure you want to delete this Dockstore {{ term }}?</h1>
+<div mat-dialog-content>
+  <app-alert></app-alert>
+  <p>You are about to permanently delete this {{ term }}. After you delete it, it will no longer appear on Dockstore.</p>
+  <p>A {{ term }} can be deleted if it:</p>
+  <ul>
+    <li>Has never been published,</li>
+    <li>Was created on/after April 1, 2022, and</li>
+    <li>Is not a checked or checker workflow.</li>
+  </ul>
+
+  <p>
+    After you delete the {{ term }}, you must uninstall the Dockstore GitHub App from the source GitHub repo, or edit/remove the
```

Comment:
~~I forget the details of your webservice PR, are we only allowing deletion of GitHub App entries? If yes, never mind; if no, this text may not always be applicable.~~

OK, read your PR description after I entered the above... Do we bring up a different dialog for non-GitHub app entries? Does it look radically different?

Reply:
For classic tools, the "Delete" button is red[-ish], and we display a much more spartan dialog:
<img width="569" alt="Screen Shot 2023-07-26 at 2 17 14 PM" src="https://github.com/dockstore/dockstore-ui2/assets/88108675/c11383f0-e04e-4a41-99a5-60f942f76772">

Reply:
(The new delete functionality in this PR will apply to all derivates of `Workflow`, not matter how they were registered.)